modules/oadp-release-notes-1-2-0.adoc

```diff
@@ -27,52 +27,9 @@ The OADP Data Mover enables you to back up Container Storage Interface (CSI) vol
 include::snippets/technology-preview.adoc[]
 
 [id="fixed-bugs-1-2-0_{context}"]
-== Fixed bugs
+== Resolved issues
 
-The following bugs have been fixed in this release:
-
-* link:https://issues.redhat.com/browse/OADP-144[OADP-144]
-* link:https://issues.redhat.com/browse/OADP-639[OADP-639]
-* link:https://issues.redhat.com/browse/OADP-1741[OADP-1741]
-* link:https://issues.redhat.com/browse/OADP-1152[OADP-1152]
-* link:https://issues.redhat.com/browse/OADP-1143[OADP-1143]
-* link:https://issues.redhat.com/browse/OADP-1931[OADP-1931]
-* link:https://issues.redhat.com/browse/OADP-148[OADP-148]
-* link:https://issues.redhat.com/browse/OADP-1067[OADP-1067]
-* link:https://issues.redhat.com/browse/OADP-1332[OADP-1332]
-* link:https://issues.redhat.com/browse/OADP-1164[OADP-1164]
-* link:https://issues.redhat.com/browse/OADP-1105[OADP-1105]
-* link:https://issues.redhat.com/browse/OADP-2009[OADP-2009]
-* link:https://issues.redhat.com/browse/OADP-1370[OADP-1370]
-* link:https://issues.redhat.com/browse/OADP-969[OADP-969]
-* link:https://issues.redhat.com/browse/OADP-1672[OADP-1672]
-* link:https://issues.redhat.com/browse/OADP-1151[OADP-1151]
-* link:https://issues.redhat.com/browse/OADP-988[OADP-988]
-* link:https://issues.redhat.com/browse/OADP-1941[OADP-1941]
-* link:https://issues.redhat.com/browse/OADP-1830[OADP-1830]
-* link:https://issues.redhat.com/browse/OADP-1821[OADP-1821]
-* link:https://issues.redhat.com/browse/OADP-1783[OADP-1783]
-* link:https://issues.redhat.com/browse/OADP-1719[OADP-1719]
-* link:https://issues.redhat.com/browse/OADP-1833[OADP-1833]
-* link:https://issues.redhat.com/browse/OADP-1872[OADP-1872]
-* link:https://issues.redhat.com/browse/OADP-2047[OADP-2047]
-* link:https://issues.redhat.com/browse/OADP-1932[OADP-1932]
-* link:https://issues.redhat.com/browse/OADP-1844[OADP-1844]
-* link:https://issues.redhat.com/browse/OADP-1182[OADP-1182]
-* link:https://issues.redhat.com/browse/OADP-1183[OADP-1183]
-* link:https://issues.redhat.com/browse/OADP-1798[OADP-1798]
-* link:https://issues.redhat.com/browse/OADP-1726[OADP-1726]
-* link:https://issues.redhat.com/browse/OADP-821[OADP-821]
-* link:https://issues.redhat.com/browse/OADP-1833[OADP-1781]
-* link:https://issues.redhat.com/browse/OADP-697[OADP-697]
-* link:https://issues.redhat.com/browse/OADP-1281[OADP-1281]
-* link:https://issues.redhat.com/browse/OADP-1077[OADP-1077]
-* link:https://issues.redhat.com/browse/OADP-1076[OADP-1076]
-* link:https://issues.redhat.com/browse/OADP-1670[OADP-1670]
-* link:https://issues.redhat.com/browse/OADP-1307[OADP-1307]
-* link:https://issues.redhat.com/browse/OADP-1640[OADP-1640]
-* link:https://issues.redhat.com/browse/OADP-1987[OADP-1987]
-* link:https://issues.redhat.com/browse/OADP-1934[OADP-1934]
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12418878[OADP 1.2.0 resolved issues] in Jira.
```

Comment:
this link gives me an empty filter, even when i'm logged in. is this expected?

Reply:
Sorry forgot to set public permissions on the filter. have done now
(Top right - not logged in and filter is visible)
![image](https://github.com/openshift/openshift-docs/assets/106804821/394f5894-48ff-474f-b9cc-52644d84b38d)